File: csharp/217_Contains_Duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        HashSet<int> m = new HashSet<int>();

        foreach (int num in nums) {
            if (m.Contains(num))
                return true;
            else
                m.Add(num);
        }

        return false;
    }
}

File: cpp/217_Contains_Duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> m;
        for (int num : nums) {
            if (m.contains(num)) {
                return true;
            } else {
                m.insert(num);
            }
        }
        return false;
    }
};